core/src/main/resources/error/error-classes.json

```diff
@@ -785,6 +779,13 @@
       "Malformed Protobuf messages are detected in message deserialization. Parse Mode: <failFastMode>. To process malformed protobuf message as null result, try setting the option 'mode' as 'PERMISSIVE'."
     ]
   },
+  "MISSING_AGGREGATION" : {
+    "message" : [
+      "The non-aggregating expression <expression> is based on columns which are not participating in the GROUP BY clause.",
+      "Add the columns or the expression to the GROUP BY, aggregate the expression, or use \"any_value(<expression>)\" if you do not care which of the values within a group is returned."
```

Comment:
`expression` is quoted already, correct? How this `\"any_value(<expression>)\"` looks like `"any_value("...")"`?

Reply:
Yeah, it looks like the example below:

```
[MISSING_AGGREGATION] The non-aggregating expression "c2" is based on columns which are not participating in the GROUP BY clause.
Add the columns or the expression to the GROUP BY, aggregate the expression, or use "any_value("c2")" if you do not care which of the values within a group is returned.;
```

Can we fix it to ``` `any_value("...")` ``` or something ?

Reply:
How about just leave `expression` in the error format:
```
... or use <expression> if you do not care which ...
```
and pass the message parameter:
```scala
    new AnalysisException(
      errorClass = "MISSING_AGGREGATION",
      messageParameters = Map("expression" -> toSQLExpr(new AnyValue(expression)))
```

Reply:
Cool. Let me update soon.

Reply:
I just tried your suggestion, and I checked the first <expression> one is also affected, so it looks a bit wired IMO.
For example, the error message will be shown like:

```
... The non-aggregating expression "anyvalue(c2)" is based on columns ...
... aggregate the expression, or use "any_value(c2)" if you do not care ...
```

Maybe should we separate the parameters into 2 parts like <expression> and <expression_any_value> ??

Reply:
Or maybe we simply recommend to use the `any_value()` function as below:

```
... aggregate the expression, or use `any_value()` for <expression> if you do not care ...
```

something like that?

Reply:
> "anyvalue(c2)" .. "any_value(c2)"

hmm, could you open a PR and override `prettyName` in `AnyValue`, and pass `prettyName` to `FirstLast.validateIgnoreNullExpr()`. Also, cc @vitaliili-db

Reply:
@itholic Since the PR has been merged, I am waiting for when you update your PR.